pkg/apply/merge.go

```diff
@@ -178,20 +178,23 @@ func mergeWebhookConfiguration(current, updated *unstructured.Unstructured) erro
 			continue
 		}
 
-		// If there is caBundle set in the current config, we want to keep that value
-		caBundle, found, err := unstructured.NestedString(currentWebhook, "clientConfig", "caBundle")
+		currentCABundle, currentCABundleFound, err := unstructured.NestedString(currentWebhook, "clientConfig", "caBundle")
```

Comment:
The commit message suggests this patch is only testing stuff. But you are also fixing bugs in the code. Please express it in the commit message. Say what was wrong and how is it fixed.